docs: further guidelines around adding extra scripts and assets (#19)
grafana/killercoda: docs/transformer.md

1. In each repository, commit your changes, push your branch, and open a pull request.

1. A Killercoda maintainer reviews the PR to ensure that the generate tutorial works as expected.

> 
> 
> ## Scripts and Extra Course Files

Reviewer: Our style encourages sentence case: https://developers.google.com/style/headings

```suggestion

## Scripts and extra course files
```

If your tutorial requires scripts or extra files, make sure to manually add them to the tutorial directory in the Killercoda repository. For example, if your tutorial requires a script to run:

> 1. Add the bash script to the tutorial directory in the Killercoda repository. See the [what-is-loki](../loki/what-is-loki/) tutorial for an example.

Reviewer: Prefer "refer to" over "see": https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/ReferTo.yml

```suggestion
1. Add the bash script to the tutorial directory in the Killercoda repository. Refer to the [what-is-loki](../loki/what-is-loki/) tutorial for an example.
```

> 2. Add the script to the `scripts` field in the `index.json` file:

Reviewer: You can use repetitive numbering and let the renderer auto increment for you which makes it easier to update steps: https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists

I'm a bit confused by the example because it doesn't reference the `scripts` field. JSON is very hard to demonstrate in the docs though and you have to be very verbose to help a user: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#partial-code-examples so I understand there may not be much you can do

Author: This is really good point let me flesh this out a little better!

   ```json
   "intro": {
      "text": "intro.md",
      "foreground": "setup.sh"
      },
   ```

For extra assets, such as images or configuration files:
1. Create a directory called `assets` in the tutorial directory.
2. Add the assets to the `assets` directory.
3. Add the mount path to the `index.json` file:
   ```json
    "assets": {
        "host01": [
          {"file": "/assets", "target": "/assets"}
        ]
      }
   ```

>    See the [grafana-basics](../grafana/grafana-basics/index.json) tutorial for an example.

Reviewer: ```suggestion
   Refer to the [grafana-basics](../grafana/grafana-basics/index.json) tutorial for an example.
```

If your tutorial requires scripts or extra files, make sure to manually add them to the tutorial directory in the Killercoda repository. For example, if your tutorial requires a script to run:

1. Add the bash script to the tutorial directory in the Killercoda repository. Refer to the [what-is-loki](../loki/what-is-loki/) tutorial for an example.

> 2. Add the script to the desired step within the `index.json` file. Note that `foreground` scripts run in the foreground (seen in terminal), and `background` scripts run in the background (Run in background thread). For example:

Reviewer: nit: 

```suggestion
2. Add the script to the desired step within the `index.json` file. Note that `foreground` scripts run in the foreground (seen in terminal), and `background` scripts run in the background (run in background thread). For example:
```
